R/apply_delay.R

```diff
@@ -1,7 +1,12 @@
 #' Apply the delay to generate a point nowcast
 #' Generate a point estimate of a completed reporting square (or rectangle)
 #'   from a reporting triangle that we want to complete with a nowcast and a
-#'   delay PMF. This code was adapted from code written (under an MIT license)
+#'   delay PMF. Each element is computed by taking the product of the expected
```

Comment:
I don't think it's good practice to add placeholders. This should be a new issue.

Reply:
placeholder was removed